apps/aecore/src/aec_conductor.erl

```diff
@@ -209,8 +205,8 @@ handle_call(stop_mining,_From, State = #state{ consensus = Cons }) ->
     epoch_mining:info("Mining stopped"),
     [ aec_tx_pool:garbage_collect() || is_record(Cons, consensus) andalso Cons#consensus.leader ],
     State1 = kill_all_workers(State),
-    {reply, ok, State1#state{mining_state = 'stopped',
-                             key_block_candidate = undefined}};
+    State2 = State1#state{mining_state = stopped},
```

Comment:
I think you want to set `key_block_candidate = undefined` here or else you will return a stale key-block while creating the new one?